FactorioBlueprintStringRenderer/src/com/demod/fbsr/EntityRendererFactory.java

```diff
@@ -135,7 +138,7 @@ public static void registerFactories(Consumer<EntityRendererFactory> register, F
 			for (String entityName : jsonGroup.keySet().stream().sorted().collect(Collectors.toList())) {
 				Optional<EntityPrototype> entity = table.getEntity(entityName);
 				if (!entity.isPresent()) {
-					System.err.println("MISSING ENTITY: " + entityName);
+					LOGGER.warn("MISSING ENTITY: {}", entityName);

```

Comment:
I manually picked LOGGER.warn for System.err logging without exceptions, and kept LOGGER.error if there is an exception.